app/src/main/java/org/sopt/study/mvvmstudy/ContactAdapter.kt

```diff
@@ -0,0 +1,54 @@
+package org.sopt.study.mvvmstudy
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.study.mvvmstudy.data.Contact
+import org.sopt.study.mvvmstudy.databinding.ItemContactBinding
+
+class ContactAdapter(val contactItemClick: (Contact)->Unit, val contactItemLongClick:(Contact)->Unit) : RecyclerView.Adapter<ContactAdapter.ContactViewHolder>() {
+    private var contacts:List<Contact> = listOf()
+
+    inner class ContactViewHolder(private val binding: ItemContactBinding):RecyclerView.ViewHolder(binding.root){
+//        private val tv_profile = itemView.findViewById<TextView>(R.id.item_tv_profile)
+//        private val tv_name = itemView.findViewById<TextView>(R.id.item_tv_name)
+//        private val tv_number = itemView.findViewById<TextView>(R.id.item_tv_number)
+
+        fun bind(contact:Contact){
+            binding.itemTvName.text = contact.name
+            binding.itemTvNumber.text = contact.number
+            binding.itemTvProfile.text = contact.profile.toString()
+
+            itemView.setOnClickListener{
+                contactItemClick(contact)
+            }
+
+            itemView.setOnLongClickListener{
+                contactItemLongClick(contact)
+                true
+            }
+        }
+    }
+
+    fun setContacts(contacts:List<Contact>){
+        this.contacts = contacts
+        notifyDataSetChanged()
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ContactViewHolder {
+        //val view = LayoutInflater.from(parent.context).inflate(R.layout.item_contact, parent, false)
+        val binding = ItemContactBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+        //return ContactViewHolder(view)
+        return ContactViewHolder(binding)
+    }
+
+    override fun onBindViewHolder(holder: ContactViewHolder, position: Int) {
+        holder.bind(contacts[position])
+    }
+
+    override fun getItemCount(): Int {
+        return contacts.size
+    }
```

Comment:
```suggestion
    override fun getItemCount() = contacts.size
```
한 줄만에 리턴된다면 이런식으로 간결하게 표현가능~👀👀